README.md

```
@@ -1,9 +1,9 @@
-## Boilerplate project for Respo
+# Fungi Collection
 
-```bash
-yarn build
-```
+### Workflow
 
-## License
+https://github.com/Respo/respo-moonbit-workflow
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)